packages/shared-ux/chrome/serverless_projects_documentation.md

```diff
@@ -0,0 +1,289 @@
+# Serverless project navigation
+
+## Introduction
+
+Welcome to the documentation for the serverless project navigation. We've exposed the tools to assist teams in building the navigation for their serverless projects.
+
+- [Serverless project navigation](#serverless-project-navigation)
+  - [Left Side Navigation](#left-side-navigation)
+    - [Navigation Tree Definition](#navigation-tree-definition)
+      - [Example](#example)
+      - [Navigation tree API](#navigation-tree-api)
+        - [`NavigationTreeDefinition`](#navigationtreedefinition)
+        - [`GroupDefinition`](#groupdefinition)
+        - [`RecentlyAccessedDefinition`](#recentlyaccesseddefinition)
+    - [React components](#react-components)
+      - [`unstyled`](#unstyled)
+  - [Breadcrumbs](#breadcrumbs)
+  - [Toolbar](#toolbar)
+  - [Search bar](#search-bar)
+
+### Building Blocks
+
+The serverless navigation is composed of several key building blocks that work together to form a comprehensive navigation system. These building blocks include:
+
+1. **Left Side Navigation**: Allow users to navigate through different sections of the project.
+2. **Breadcrumbs**: A visual representation of the user's current location within the navigation hierarchy.
+3. **Toolbar**: A customizable toolbar that provides quick access to important actions and features.
+4. **Search Bar**: A search input field that enables users to quickly access specific content within the project.
+
+In the following sections, we will explore each of these building blocks in detail.
+
+## Left Side Navigation
+
+The left side navigation serves as a primary means for users to navigate through different sections of the project. There are two approaches to building the side navigation:
```

Comment:
note to self: I think we should also explain the design of the nav. I think we should explain that there are predefined nav sections and pieces that we provide that can be enabled / repositioned and also that they